app/views/announcements/index.html.erb

```diff
@@ -1,19 +1,22 @@
 <!-- Tab navbar -->
-<div class="sticky-top">
+<div class="sticky-top bg-light">
   <!-- General Nav bar -->
   <nav class="navbar navbar-expand-lg navbar-light bg-dark p-0">
-    <div class="container-fluid">
-      <a class="navbar-brand" href="#">
-        <%= image_tag "bsfc_logo.png", {:alt => "Berkeley Student Food Collective Logo", :width => 40, :height => 40} %>
+    <div class = "container-fluid">
+      <a class = "navbar-brand" href = "announcements">
```

Comment:
this has been happening a lot.  NO spaces around equals signs in HTML.  please correct throughout